doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -2461,3 +2461,75 @@ Example of QUIC logging with a CYU hash:
         }
     ]
   }
+
+Event type: DHCP
+-----------------
+
+The default logging level only logs DHCP reply types which contains the
+MAC address and assigned IP of requesting clients.
+
+Fields
+~~~~~~
+
+* "type": DHCP message type
```

Comment:
Maybe remove the `DHCP` bit here, and add the possible types (`request`, `reply` or `<unknown`>), to try to reduce confusion of having two different fields with the same explanation?